### PR TITLE
All: Run fixes for prod use

### PR DESCRIPTION
**Pull request by rssk**
# Problem
Cstac wasnt ready for production deployment
# Solution
Fixed by removing test data as well as cleaning up ports/routing to make reverse proxying easier

# Testing
Fire up cstac _without_ a local config and run SSR

### CODE REVIEW

File: packages/coinstac-client-core/src/index.js [hunk 1]
```diff
@@ -214,11 +216,7 @@ class CoinstacClient {
 
       for (let i = 0; i < filesArray.length; i += 1) {
         linkPromises.push(
-          fs.link(
-            filesArray[i],
-            `${this.appDirectory}/${this.clientId}/${runId}/${filesArray[i].replace(/\//g, '-')}`,
-            err => console.log(err) // eslint-disable-line no-console
-          )
+          linkAsync(filesArray[i], `${this.appDirectory}/${this.clientId}/${runId}/${filesArray[i].replace(/\//g, '-')}`)
```
Comment (rssk) [inline packages/coinstac-client-core/src/index.js:220]: Check any other instances of this if you can

File: packages/coinstac-server/src/routes/pipeline.js [hunk 2]
```diff
@@ -90,6 +90,7 @@ module.exports = [
               saveResults(run.id, result);
             })
             .catch((error) => {
+              console.log(error);
```
Comment (rssk) [inline packages/coinstac-server/src/routes/pipeline.js:93]: for now, makes debugging server side tasks 100% easier

File: packages/coinstac-ui/.ignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+app/render/build/
```
Comment (rssk) [inline packages/coinstac-ui/.ignore:1]: This is an `ag` ignore to make searching sane

File: packages/coinstac-ui/app/render/components/dashboard/dashboard.jsx [hunk 4]
```diff
@@ -54,8 +54,8 @@ class Dashboard extends Component {
     const { router } = this.context;
 
     // Uncomment to clear local data
-    this.props.clearRuns();
-    this.props.initTestData();
+    // this.props.clearRuns();
```
Comment (rssk) [inline packages/coinstac-ui/app/render/components/dashboard/dashboard.jsx:57]: Be really great to have another way to do this not in the UI code

File: packages/coinstac-ui/app/render/state/ducks/collections.js [hunk 5]
```diff
@@ -14,7 +15,7 @@ const GET_ASSOCIATED_CONSORTIA = 'GET_ASSOCIATED_CONSORTIA';
 const REMOVE_COLLECTIONS_FROM_CONS = 'REMOVE_COLLECTIONS_FROM_CONS';
 const SET_COLLECTIONS = 'GET_ALL_COLLECTIONS';
 
-function iteratePipelineSteps(consortium, filesByGroup) {
+function iteratePipelineSteps(consortium, filesByGroup, baseDirectory) {
```
Comment (rssk) [inline packages/coinstac-ui/app/render/state/ducks/collections.js:18]: This block was changed the most, it now replaces the paths in the first csv column with the full path using the csv dir as a base, this works for our demo files, but maybe not others.

File: packages/coinstac-client-core/src/index.js (hunk 1 above)
Comment (tkah) [reply packages/coinstac-client-core/src/index.js:220]: The same issue is likely found in the `unlinkFiles` function. I'll take a look.

File: packages/coinstac-ui/app/render/components/dashboard/dashboard.jsx (hunk 4 above)
Comment (tkah) [reply packages/coinstac-ui/app/render/components/dashboard/dashboard.jsx:57]: Issue #405 is asking for a solution to this. I'm thinking maybe a button in the UI to reset? Could be in dev mode only.

**Review by tkah — APPROVED** [review]
Looks good

**Pull request merged**